bin/lcov

```diff
@@ -159,6 +159,7 @@ our $cwd = `pwd`;	# Current working directory
 our $data_stdout;	# If set, indicates that data is written to stdout
 our $follow;		# If set, indicates that find shall follow links
 our $diff_path = "";	# Path removed from tracefile when applying diff
+our $fail_under_lines = 0;
```

Comment:
Please rename to `opt_fail_under_lines` to identify this variable as the result of a command line option.

Reply:
Done.